apps/src/templates/sectionProgressV2/ExpandedProgressDataColumn.jsx

```diff
@@ -119,12 +98,7 @@ function ExpandedProgressDataColumn({
       <caption hidden={true}>
         {i18n.progressForLesson({lessonName: lesson.title})}
       </caption>
-      <ExpandedProgressColumnHeader
-        lesson={lesson}
-        removeExpandedLesson={removeExpandedLesson}
-        expandedChoiceLevels={expandedChoiceLevels}
-        toggleExpandedChoiceLevel={toggleExpandedChoiceLevel}
-      />
+      <ExpandedProgressColumnHeader lesson={lesson} />
```

Comment:
This is a great example of why I wanted to do this. Dramatically reduces the number of props that are just passed through.